### ISSUE TITLE
large model release, would you update it?

### ISSUE THREAD

**Issue by chikiuso** [polakowo/gpt2bot#8]
hi I see the large model of gpt2 and the pretrained dialogpt based on it is released, would you consider include in this project? thanks!

**Comment by polakowo** [comment]
Yes, I'm going to update the package to the new transformers version soon.

**Comment by chikiuso** [comment]
thanks a lot :D

**Comment by x0rzkov** [comment]
Yeah, that would be awesome !

**Comment by polakowo** [comment]
I released a new version that accepts any text generation model, including large dialogpt (use `model = microsoft/DialoGPT-large` in chatbot.cfg).

**Comment by lucmichalski** [comment]
@polakowo 

I made a fork and added Dockerfiles with GPU/CPU support. It works awesome !

Would be awesome to use it with the [deeppavlov-agent](https://deeppavlov-agent.readthedocs.io/en/latest/)...

Thanks again !

**Comment by polakowo** [comment]
@lucmichalski looks great! I had a Dockerfile previously but it downloaded the same model every time I deployed the container, which made it too expensive to play with. Maybe I should have synced caching dirs to avoid this.

**Comment by lucmichalski** [comment]
@polakowo 

do you want me to do a PR ? I used a docker volume for the model
